test/splitting/annotations.unit

```diff
@@ -0,0 +1,9 @@
+40 columns                              |
+>>> blet
+@veryLongPrefix.VeryLongClassName<VeryLongTypeArgument, OtherVeryLongTypeArgument>(veryLongArgument, otherVeryLongArgument, thirdVeryLongArgument) int i = 0;
+<<<
+@veryLongPrefix.VeryLongClassName<VeryLongTypeArgument, OtherVeryLongTypeArgument>(
+    veryLongArgument,
+    otherVeryLongArgument,
+    thirdVeryLongArgument)
+int i = 0;
```

Comment:
It would be good to have some examples of nested generics too, and how splitting and indentation is handled in them. You can copy the snippets from splitting/type_arguments.stmt and tweak them to be annotations.

Reply:
Will do.